### PR TITLE
Add branch naming rules and prefixes

### PR DESCRIPTION
**Pull request by roy-pon**
Rules and prefixes added to the branching guidelines

### CODE REVIEW

**Review by pon-james — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -39,7 +39,31 @@ setup separate feature branches
 
 ### Branches
 
-Branch names clearly communicate purpose and should contain a reference to issue tracking system if relevant
+***Basic Rules***
+
+**1. Lowercase and hyphen-separated:** Keep branch names in lowercase and separate words with hyphens. For example, use `feature/booking-tool` or `bugfix/cookie-banner-refresh`.
+
+**2. Alphanumeric characters:** Use only alphanumeric characters (letters a-z, numbers 0–9) and hyphens. Avoid using punctuation, spaces, underscores, or any other non-alphanumeric characters.
```
Comment (zeger-pon) [inline README.md]: I would prefer to make it even more strict, not avoid but do not use. And for fun add the regex (feature|bugfix|hotfix|docs|chore)\/[a-z0-9]+(-([a-z0-9])+)*[a-z0-9]+

**Review by danielkarso-pon — APPROVED** [review]

**Review by zeger-pon — CHANGES_REQUESTED** [review]

**Comment by apiest-pon** [comment]
Good practice starts by leading by example, why don't you use it yourself?

Spoke too soon, new prefixes 😛

**Review by apiest-pon — APPROVED** [review]

**Review by zeger-pon — APPROVED** [review]

**Review by pon-james — APPROVED** [review]
LGTM

**Comment by roy-pon** [comment]
@zeger-pon small reminder for this PR, think it can be merged. Only you can do this as a owner.

**Pull request merged**